<?xml version="1.0"?>
<launch>
    <arg name="object_name" default="HV8" />
  <arg name="icpRefine" default="False" />
  <arg name="arch" default="JSIS3D" />
  <arg name="process_swich" default="object_segment" />
  <arg name="resolution" default="8192" />
  <arg name="load_path" default="/home/ericlab/DENSO_results/August/semantic_segmentation/ishiyama/PointNet_Segmentation/semantic_changed_occulution_kiriwake_9_14_1000_1.hdf5/latest_net.pth"/>
  <node name="random_move" pkg="gen_dataset" type="random_move_euler.py">
    <param name="object_name" value="$(arg object_name)" />
  </node>
  <node name="estimator_client" pkg="estimator" type="estimator_client.py" output="screen" >
    <param name="object_name" value="$(arg object_name)" />
    <param name="icpRefine" value="$(arg icpRefine)" />
    <param name="arch" value="$(arg arch)" />
    <param name="resolution" value="$(arg resolution)" />
  </node>
</launch>